metadata_service/proxy/atlas_proxy.py

```diff
@@ -35,6 +35,11 @@
 _ATLAS_PROXY_CACHE_EXPIRY_SEC = 11 * 60 * 60 + randint(0, 3600)
 
 
+class STATUS:
```

Comment:
please use camelcase and maybe subclass Enum ?

Reply:
Oh nice catch, it should be Title Case.